tribomobile/App.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React from 'react';
+import {
+  SafeAreaView,
+  Text,
+  StatusBar,
+  View,
+} from 'react-native';
+import SplashScreen from './screens/splash'
+
+const App: () => React$Node = () => {
+  return (
+    <>
+        <SplashScreen/>
+    </>
```

Comment:
I suggest using 2 spaces for indentation instead 4 spaces.
```suggestion
    <>
      <SplashScreen/>
    </>
```